pkg/reconciler/v1alpha1/broker/broker_test.go

```diff
@@ -83,971 +92,920 @@ func init() {
-			},
-			Mocks: controllertesting.Mocks{
-				MockGets: []controllertesting.MockGet{
-					func(_ client.Client, _ context.Context, key client.ObjectKey, obj runtime.Object) (controllertesting.MockHandled, error) {
-						if _, ok := obj.(*appsv1.Deployment); ok {
-							if strings.Contains(key.Name, "ingress") {
-								return controllertesting.Handled, errors.New("test error getting ingress Deployment")
-							}
-						}
-						return controllertesting.Unhandled, nil
-					},
-				},
-			},
-			WantEvent:  []corev1.Event{events[brokerReconcileError]},
-			WantErrMsg: "test error getting ingress Deployment",
+			Name: "Broker not found",
+			Key:  testKey,
 		},
 		{
-			Name:   "Ingress Deployment.Create error",
-			Scheme: scheme.Scheme,
-			InitialState: []runtime.Object{
-				makeBroker(),
-				makeTriggerChannel(),
-			},
-			Mocks: controllertesting.Mocks{
-				MockCreates: []controllertesting.MockCreate{
-					func(_ client.Client, _ context.Context, obj runtime.Object) (controllertesting.MockHandled, error) {
-						if d, ok := obj.(*appsv1.Deployment); ok {
-							if d.Labels["eventing.knative.dev/brokerRole"] == "ingress" {
-								return controllertesting.Handled, errors.New("test error creating ingress Deployment")
-							}
-						}
-						return controllertesting.Unhandled, nil
-					},
-				},
+			Name: "Broker is being deleted",
+			Key:  testKey,
+			Objects: []runtime.Object{
+				testing2.NewBroker(brokerName, testNS,
+					testing2.WithBrokerChannelProvisioner(channelProvisioner("my-provisioner")),
+					testing2.WithInitBrokerConditions,
+					testing2.WithBrokerDeletionTimestamp),
 			},
-			WantEvent:  []corev1.Event{events[brokerReconcileError]},
-			WantErrMsg: "test error creating ingress Deployment",
 		},
 		{
-			Name:   "Ingress Deployment.Update error",
-			Scheme: scheme.Scheme,
-			InitialState: []runtime.Object{
-				makeBroker(),
-				makeTriggerChannel(),
-				makeDifferentIngressDeployment(),
-			},
-			Mocks: controllertesting.Mocks{
-				MockUpdates: []controllertesting.MockUpdate{
-					func(_ client.Client, _ context.Context, obj runtime.Object) (controllertesting.MockHandled, error) {
-						if d, ok := obj.(*appsv1.Deployment); ok {
-							if d.Labels["eventing.knative.dev/brokerRole"] == "ingress" {
-								return controllertesting.Handled, errors.New("test error updating ingress Deployment")
-							}
-						}
-						return controllertesting.Unhandled, nil
-					},
-				},
-			},
-			WantEvent:  []corev1.Event{events[brokerReconcileError]},
-			WantErrMsg: "test error updating ingress Deployment",
+			Name: "Trigger Channel.Create error",
+			Key:  testKey,
+			Objects: []runtime.Object{
+				testing2.NewBroker(brokerName, testNS,
+					testing2.WithBrokerChannelProvisioner(channelProvisioner("my-provisioner")),
+					testing2.WithInitBrokerConditions),
+			},
+			WantCreates: []metav1.Object{
+				testing2.NewChannel("", testNS,
+					testing2.WithChannelGenerateName(channelGenerateName),
```

Comment:
you can get rid of all `testing2.` by doing the comment in the imports.

Reply:
done!